code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -459,40 +459,55 @@
 		return TRUE
 
 /obj/item/clothing/accessory/holobadge/attack(mob/living/carbon/human/H, mob/living/user)
-	if(isliving(user))
+	if(H != user)
 		user.visible_message("<span class='warning'>[user] invades [H]'s personal space, thrusting [src] into [H.p_their()] face insistently.</span>",
-		"<span class='warning'>You invade [H]'s personal space, thrusting [src] into [H.p_their()] face insistently. You are the law.</span>")
+		"<span class='warning'>You invade [H]'s personal space, thrusting [src] into [H.p_their()] face insistently. You are THE LAW!</span>")
+		return
+	..()
```

Comment:
This is fine now, but doesn't really have anything to do with the main idea of the PR